docs/_data/versions.yml

```
@@ -17,6 +18,7 @@ spec:
       name: Version 0.1
```

I'd call this stable, no?

```suggestion
      name: Version 0.1
      status: Stable
```

---

As said earlier, I think that's reasonable given that it is in use.